### ISSUE TITLE
iframe on the home page missing a title

### ISSUE THREAD

**Issue by christosptr** [mr-bobz/talarts#99]
Code with the issue
<iframe width="960" height="570" frameborder="0" src="http://www.youtube.com/embed/FprQKc0bCKY?fs=1&amp;showsearch=0&amp;showinfo=1&amp;iv_load_policy=1&amp;wmode=transparent&amp;controls=2&amp;disablekb=1&amp;modestbranding=1&amp;rel=0" aria-label="Promotional Video" class="youtube-player">TALA promotional video by Pitch Productions</iframe>

Suggested Solution
<iframe title="TALA promotional video by Pitch Productions" width="960" height="570" frameborder="0" src="http://www.youtube.com/embed/FprQKc0bCKY?fs=1&amp;showsearch=0&amp;showinfo=1&amp;iv_load_policy=1&amp;wmode=transparent&amp;controls=2&amp;disablekb=1&amp;modestbranding=1&amp;rel=0" aria-label="Promotional Video" class="youtube-player">TALA promotional video by Pitch Productions</iframe>

**Comment by HerinHentry** [comment]
@christosptr : Duplicate issue #42

**Comment by HerinHentry** [comment]
http://talarts.knowbility.org/
The Title attribute has been completely removed. This is an accessibility violation. Please include Title attribute. #42 was related to this and looks like this was removed as a fix for 42.

**Comment by christosptr** [comment]
@ herin when I check the page in Wordpress I can see the title but when I inspect the code on the page I do not see the title of the video. Not sure what is going on here
@mr-bobz do you have any idea what is going on here?

![image](https://cloud.githubusercontent.com/assets/5661003/11020507/c88d00b8-8674-11e5-9e18-fd19d76db593.png)

**Comment by mr-bobz** [comment]
As mentioned in #42, you tube seems to be removing the title we set.

Anyone interested to try this plugin instead of youtube?
https://wordpress.org/plugins/accessible-video-library/

**Comment by mr-bobz** [comment]
iframe removed
